sed/__init__.py

```diff
@@ -1,7 +1,7 @@
 """sed module easy access APIs
 
 """
-from .core.processor import SedProcessor
+# from .core.processor import SedProcessor
 
 __version__ = "0.1.0"
-__all__ = ["SedProcessor"]
+# __all__ = ["SedProcessor"]
```

Comment:
Why do you want to remove this? This is typically the only thing you want to import and use (right now).

Reply:
I also have this always commented out but the reason is that it forces importing everything, due to the current structure of the processor. On the long run, this should be there though

Reply:
I re-enabled this here. If we want to disable this, let's have a discussion on this, and a separate PR.